src/apps/concepts/components/ViewConceptsHeader.tsx

```diff
@@ -126,20 +130,31 @@ const ViewConceptsHeader: React.FC<Props> = ({
           open={Boolean(switchSourceAnchor)}
           onClose={handleSwitchSourceClose}
         >
-          <TextField 
-          multiline
-          className={classes.textField} 
-          InputProps={{
-            className: classes.underline
-          }} 
-            inputProps={{
-              className: classes.input
-            }}
-            value={
-              showSources ? "Choose a source/dictionary" : "Select a different source/dictionary"
-            }
-            onClick={() => setShowSources(!showSources)}
-          />
+        {/* <Button 
+           variant="text" 
+           onClick={() => setShowSources(!showSources)}
+          > 
+             {showSources ? "Select a recommended source" : "Select an alternative source"}
+        </Button> */}
+        <Input
+          // onChange={e => setQ(e.target.value)}
+          // value={q}
+          color="secondary"
+          type="search"
+          fullWidth
+          placeholder={`choose a source`}
+          data-testid="searchInput"
+          endAdornment={
+            <InputAdornment position="end">
+              <IconButton
+                onClick={() => setShowSources(!showSources)}
+                data-testid="searchButton"
+              >
+                <SearchIcon />
+              </IconButton>
+            </InputAdornment>
+          }
+        />
```

Comment:
@jwnasambu I can see where you're going here, but we really do want two independent elements that have relatively independent functions. Part of this is because it's very counter-intuitive (I think) to switch the list based on the icon in the search bar when that has... no obvious connection to the two lists displayed.

The `onClick` handler in the `IconButton` should trigger a search against the current list of sources (and we should only display the search bar when looking at the alternative sources, not the default sources.

Reply:
I mentioned on Talk, I think, that we should use a `Grid` component to keep everything here orientated in a single column. So I would wrap both the Button and the Input in an element like this:

```ts
<Grid
  container
  direction="column">
// content
</Grid>
```

Reply:
@ibacher kindly ignore my current changes for I made without reading through your comment.